packages/web-app-files/src/views/FilesDrop.vue

```diff
@@ -11,7 +11,7 @@
       <div v-else key="loaded-drop" class="oc-flex oc-flex-column oc-flex-middle oc-height-1-1">
         <div class="oc-text-center oc-width-1-1 oc-width-xxlarge@m">
           <h2 v-text="title" />
-          <file-upload
+          <resource-upload
             id="files-drop-zone"
             ref="fileUpload"
             class="uk-flex uk-flex-middle uk-flex-center uk-placeholder"
```

Comment:
```suggestion
            class="oc-flex oc-flex-middle oc-flex-center oc-placeholder"
```
Not sure if the oc-placeholder class is valid after all, and no idea how we still have `uk-` classes 🤔